scripts-5.8/postinstall.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+export PKG_PATH="$MIRROR/pub/OpenBSD/`uname -r`/packages/`arch -s`/"
+
+# create vagrant user/group and home directory
+groupadd vagrant
+adduser -noconfig -unencrypted -batch vagrant vagrant,wheel vagrant vagrant
```

Comment:
Not sure... we install sudo from ports now, could that cause any trouble?

What's a reliable way to check if a user exists?

If we unite the scripts into `scripts/`, I think `vagrant.sh` should go away, even for the older templates. To me it didn't make much sense to have it in a separate script, and `postinstall.sh` also did things related to Vagrant (`export PKG_PATH...`)